kernel/sorts.mli

```diff
@@ -86,6 +86,10 @@ module Quality : sig
 
   module Map : CMap.ExtS with type key = t and module Set := Set
 
+  type pattern =
+    PQProp | PQSProp | PQType | PQVar of int option
```

Comment:
```suggestion
    PQConstant of constant | PQVar of int option
```
?

Reply:
I don't know why I didn't do it from the start.